cmd/kind/load/docker-image/docker-image.go

```diff
@@ -119,7 +119,7 @@ func runE(flags *flagpole, args []string) error {
 		id, err := node.ImageID(imageName)
 		if err != nil || id != imageID {
 			selectedNodes = append(selectedNodes, node)
-			log.Debugf("Image: %q with ID %q not present on node %q", imageName, imageID, node.String())
+			globals.GetLogger().V(0).Infof("Image: %q with ID %q not present on node %q", imageName, imageID, node.String())
```

Comment:
Do we need `V(0)` here? The klog specification says that `klog.Info` and `klog.V(0).Info` are semantically equivalent afaik?

Reply:
Also, do we want to consider maybe creating an internal wrapper package or something so we can just call `log.Info` instead of `globals.GetLogger().Info` every time? It could quite simply wrap the default configured logger 😄

Reply:
our interface is slightly simplified by only having Info on V()

Reply:
yeah probably? but also we can grep for this and be like "bad globals, bad!" :P

Reply:
in a lot of these places we could have also stored the return of `GetLogger()` but this was a pretty mechanical port (read: lots of sed)